<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShelfBooks</class>
 <widget class="QWidget" name="ShelfBooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="shelvesButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>SHELVES</string>
   </property>
  </widget>
  <widget class="QLabel" name="shelfNameLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>481</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> label name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addToShelf">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD TO SHELF</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SHELF:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>851</width>
     <height>641</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>10</y>
     <width>391</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Search :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
